android/src/main/java/com/mr/flutter/plugin/filepicker/FileUtils.java

```diff
@@ -471,4 +465,18 @@ private static String getDocumentPathFromTreeUri(final Uri treeUri) {
         else return File.separator;
     }
 
+    private static void recursiveDeleteFile(final File file) throws Exception {
+        if (!file.exists()) {
```

Comment:
The old code had a null check on the file too. Perhaps add that here as well?

Reply:
Thanks, I fixed. https://github.com/miguelpruivo/flutter_file_picker/pull/1507/commits/375a10a6d28bb487f2dcae088b7ecf48d3f7852e